exercises/practice/grains/.docs/instructions.md

```diff
@@ -13,14 +13,15 @@ There are 64 squares on a chessboard (where square 1 has one grain, square 2 has
 
 Write code that shows:
 - how many grains were on a given square, and
-- the total number of grains on the chessboard
+- the total number of grains on the chessboard,
+- and throws a Value Error for numbers over 64.
 
 ## For bonus points
 
 Did you get the tests passing and the code clean? If you want to, these
 are some additional things you could try:
 
-- Optimize for speed.
+- Optimize for speed. Bitwise operators anyone?
```

Comment:
1. If you want to have people look into bitwise operators, a "read up on bitwise operators here `link`" might be better than a question.
2. Out of curiosity, are those actually faster or is that something that's already optimized for you by the interpretter?